app/controllers/api.js

```diff
@@ -6,85 +6,88 @@ var ErrorReport = require('../models/errorReport');
 var API = new Object();
 
 API.checkOut = function (req, res) {
-    //Get string data from card and send to other API - get studentID back
     var cardString = req.body.cardString;
     var dockID = req.body.dockID;
     var bikeID = req.body.bikeID;
-
-    //Look up if student currently has another bike out
-    Dock.findOne({ dockID: dockID}, function(err, dock) {
+	// TODO - check for already checked out bike
+	var validTransaction = true;
```

Comment:
That is correct. An invalid transaction is one where the student already has a bike checked out and we need to ignore the transaction. I was messing around with it for a bit and I thought it would be better suited to add when we do the rest of the validation stuff.